### DISCUSSION TITLE
nvtop throwing error when in Ghostty but works in other terminals

### DISCUSSION THREAD

**Discussion by zabirauf** [ghostty-org/ghostty#3816] [Q&A] [answered]
I have the following steps where the issue repros

1. Open Ghostty (which uses zsh for me)
2. SSH into a machine equipped with Nvidia GPUs
3. Run `nvtop` and it fails with `Error opening terminal: xterm-ghostty`

<img width="317" alt="Screenshot 2024-12-28 at 15 37 27" src="https://github.com/user-attachments/assets/f1528c7b-94ef-4e58-93db-414500aa2e59" />

When I use Mac Terminal, Alacrity, iTerm, it works correctly.

Mac: 15.1.1 (24B91)
Ghostty: https://github.com/ghostty-org/ghostty/commits/4b4d4062 v1.0.0

**Comment by jmckillican** [comment, marked as answer]
terminfo issue mentioned in documents with how to fix it:  https://ghostty.org/docs/help/terminfo

**Reply by zabirauf** [reply]
Thanks a lot, following the SSH instructions from the doc solved the problem.
